Symfony/CS/Tests/Fixer/Contrib/NoUselessReturnFixerTest.php

```diff
@@ -34,6 +34,44 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                    function bar($baz)
+                    {
+                        if ($baz)
+                            return $this->baz();
+                        else
```

Comment:
That is true, but it might help to identify useless or not optimal code paths that can be fixed automatically.. but indeed we are at the border between CS and SCA fixing ;)